lms/templates/registration/activate_account_notice.html

```diff
@@ -1,14 +1,21 @@
-<%! from django.utils.translation import ugettext as _ %>
+<%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import Text, HTML
```

Comment:
Nit: The docs now have this properly alphabetized as "HTML, Text".  It's up to you if you want to fix.